src/render/OpenGL.cpp

```diff
@@ -2045,7 +2045,8 @@ bool CHyprOpenGLImpl::shouldUseNewBlurOptimizations(PHLLS pLayer, PHLWINDOW pWin
     return false;
 }
 
-void CHyprOpenGLImpl::renderTextureWithBlur(SP<CTexture> tex, CBox* pBox, float a, SP<CWLSurfaceResource> pSurface, int round, bool blockBlurOptimization, float blurA) {
+void CHyprOpenGLImpl::renderTextureWithBlur(SP<CTexture> tex, CBox* pBox, float a, SP<CWLSurfaceResource> pSurface, int round, bool blockBlurOptimization, float blurA,
```

Comment:
why this change? why not just multiply it beforehand?

Reply:
Because overallA multiplies both the texture and blur texture A, regardless of PBLURIGNOREOPACITY .

Reply:
without that we might multiply by overallA^2 if !PBLURIGNOREOPACITY

Reply:
makes sens